### PR TITLE
fix(integrating-Autoware): add notes on LiDAR-SLAM

### PR DESCRIPTION
**Pull request by sfukuta**
Signed-off-by: Shigekazu Fukuta <shigekazu.fukuta@tier4.jp>

## Description

Added notes when using LiDAR-SLAM.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by sfukuta** [comment]
@kenji-miyake Request you to review the pull request.

File: docs/how-to-guides/integrating-autoware-with-your-vehicle.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Some examples are:
 - [HDL](https://github.com/koide3/hdl_graph_slam)
 - [LIO-SAM](https://github.com/TixiaoShan/LIO-SAM)
 - [LiDAR-SLAM (ROS 2)](https://github.com/rsasaki0109/lidarslam_ros2)
+  - You need to relay the autoware topic name so that LiDAR-SLAM can receive it.
```
Comment (kenji-miyake) [inline docs/how-to-guides/integrating-autoware-with-your-vehicle.md:27]: @sfukuta Could you clarify this sentence a bit more? I couldn't understand well.
I guess it's better to write some console commands instead.
cc @kminoda @shmpwk @soblin

Comment (sfukuta) [reply docs/how-to-guides/integrating-autoware-with-your-vehicle.md:27]: @kenji-miyake 
Thanks you.This sentence mean as follows.

To use LiDAR-SLAM(ROS2), I need "/tf" and "/input_cloud" as input topics.
However, "/input_cloud" is remapped to "/velodyne_points".
Autoware doesn't have "/velodyne_points" so you need to convert topics of the same type.

The topic name convert command is `ros2 run topic_tools relay <autoware topic name> /velodyne_points`

Comment (kenji-miyake) [reply docs/how-to-guides/integrating-autoware-with-your-vehicle.md:27]: @kminoda Would it be possible for you to give me some proposals for this change?

Comment (kminoda) [reply docs/how-to-guides/integrating-autoware-with-your-vehicle.md:27]: Maybe something like this? Let me know if I misunderstood your thought.


> Note that if you are using the sensing modules in Autoware when creating the pointcloud map using the above tools, make sure that the topic names are properly renamed.

Comment (kminoda) [reply docs/how-to-guides/integrating-autoware-with-your-vehicle.md:27]: But it seems to me that this side note is not about Autoware but rather about ROS 2 in general. Maybe the Autoware Documentation is not the right place to feedback the issue. 

Assuming that @.sfukuta-san had encountered above issue, how about posting that issue on Autoware Discussions?

Comment (kminoda) [reply docs/how-to-guides/integrating-autoware-with-your-vehicle.md:27]: For example like this:

Q: I am having an issue using LiDAR-SLAM ROS2 with Autoware sensing modules. I want to create pointcloud map but the SLAM doesn't start.
A: Changing the name of output sensor topics from Autoware fixed the issue. We need to rename the topics in the following procedure...

Comment (sfukuta) [reply docs/how-to-guides/integrating-autoware-with-your-vehicle.md:27]: @kminoda 
Thanks you.I'll try posting this on the Autoware Discussions.
In that case, can I post it in the Q&A category and answer it myself?

Comment (kminoda) [reply docs/how-to-guides/integrating-autoware-with-your-vehicle.md:27]: I guess it's OK as long as the issue and the solution is clear enough.

**Comment by shmpwk** [comment]
@sfukuta Still in progress?

**Comment by sfukuta** [comment]
@shmpwk It's in progress, so please wait for a while.

**Comment by sfukuta** [comment]
@shmpwk @kenji-miyake @kminoda 

It's late, but I created it in a discussion.
https://github.com/autowarefoundation/autoware/discussions/2959

**Pull request closed without merging**